### PR TITLE
Them component ao ma link

### PR DESCRIPTION
**Pull request by ngocducpham**

### CODE REVIEW

**Comment by ngocducpham** [comment]
Merge ddee

File: src/components/AoMaLink/index.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styles from './index.module.scss';
+
+function AoMaLink({ children, className, ...props }) {
+    return (
+        <a className={`${styles.link} ${className}`} {...props}>
```
Comment (betonamu) [inline src/components/AoMaLink/index.jsx]: use classnames here

Comment (ngocducpham) [reply src/components/AoMaLink/index.jsx]: i'm got it

**Comment by betonamu** [comment]
merged. thank you so much for your contribution to lor-components project.

File: src/components/AoMaLink/index.jsx (hunk 1 above)
Comment (betonamu) [reply src/components/AoMaLink/index.jsx]: no. you should say "i got it"

Comment (ngocducpham) [reply src/components/AoMaLink/index.jsx]: sorry, i got "ao ma"

**Pull request merged**